.pylintrc

```diff
@@ -2,6 +2,10 @@
 #
 #   pylint core --rcfile=.pylintrc -i y
 
+# NOTE: This file should be kept in sync with .pylintrc in
```

Comment:
Not sure this one is needed. I mean, it's already not in sync, and in fact "better" because it doesn't disable a bunch of stuff. I think it's fine for this to evolve locally with the repository.